R/run.R

```diff
@@ -191,6 +191,11 @@ log_image_to_run <- function(name, path = NULL, plot = NULL,
   if (is.null(run)) {
     run <- get_current_run()
   }
+  if (!is.null(plot)) {
+    path <- "_generated_rplot.png"
+    ggsave(filename = path, plot = plot)
```

Comment:
is ggsave from ggplot2? if so, shouldn't claim gglot2 as dependency of r sdk?

Reply:
yes, added